source/simulator/stokes_matrix_free.cc

```diff
@@ -2540,51 +2538,22 @@ namespace aspect
             additional_data.mapping_update_flags = (update_gradients | update_JxW_values |
```

Comment:
why this change from value to gradients? is it because we now need gradients and values? should we then specify both?

Reply:
Good catch. Let me clean this up.

Reply:
it turns out that the flags were incorrect anyways (it turns out _values doens't matter).